lib/pipeline/addon/components/settings-git/template.hbs

```diff
@@ -19,6 +23,25 @@
       <h1 class="pull-left">{{t 'authPage.header'}}</h1>
     </section>
 
+    <section>
+      <div class="row nav nav-boxes checked-active">
+        <a href="#" alt={{selectedOauthType}} class="col span-2 nav-box-item driver active">
+          {{#if (eq selectedOauthType 'github')}}
```

Comment:
For both this and gitlab you check the param directly and use the `eq` helper. Wouldn't it make sense to split these into `isGithub` and `isGitlab`?